js-weather/weather.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+//APIのコールを関数化
+const REQUEST_AJAX = async (city_id) => {
+
+    const URL = `http://api.openweathermap.org/data/2.5/weather?id=${city_id}&appid=4b5774e9f3d2a07b84f0f2f88e486224&lang=ja&units=metric`;
+    const JSON = await fetch(URL)
+        .then(res => {
+            return res.json();
+        }).catch(error => {
+            console.log('APIへのアクセスに失敗しました');
+            return null;
+        });
+    const WEATHER = JSON.weather[0].description;
+    const WEATHER_DESCRIPTION = document.getElementById('weather_description');
+    WEATHER_DESCRIPTION.textContent = `現在の天気は${WEATHER}です`;
+
+}
```

Comment:
(修正不要)
```suggestion
};
```
アロー関数の定義の末尾にもセミコロンがあるとよいですね！
セミコロン入れるのとかは自動整形のプラグインにやってもらいましょう。